README.md

```diff
@@ -1,2 +1,48 @@
-# sparksftpTest
-Testing different methods to read from SFTP from Spark
+## sparksftpTest is a project to showcase an ETL pipeline: SFTP - Hive - SFTP with Spark
+
+The process can be summarized in
+
+-   Ingestion of the Data (CSV) from an SFTP server.
+-   Processed by Spark and stored in Parquet/Hive, original data + the ones with tranformations
+-   Storing the results on the SFTP server.
+
+There is currently two implemenations
+
+### Spark with a SFTP connector
+
+This is one of the prototypes developed that works and is based on the springml connector https://github.com/springml/spark-sftp
+
+This solution is currently the fastest
+[ReadingSFTPConnectorApp](https://github.com/47deg/sparksftpTest/blob/master/src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPConnectorApp.scala)
```

Comment:
These paths can be relative: `/src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPConnectorApp.scala`

Reply:
Changed. Thanks